src/cmd/services/m3dbnode/main/main_index_test.go

```diff
@@ -209,6 +211,20 @@ func TestIndexEnabledServer(t *testing.T) {
 		{value: 3.0, at: start.Add(2 * time.Second), unit: xtime.Second},
 	}
 
+	// unknown ns test
+	randomNs := "random-ns"
+	for _, v := range values {
+		err := session.WriteTagged(ident.StringID(randomNs),
+			ident.StringID("foo"),
+			ident.NewTagsIterator(ident.NewTags(
+				ident.StringTag("foo", "bar"),
+				ident.StringTag("baz", "foo"),
+			)),
+			v.at, v.value, v.unit, nil)
+		require.Error(t, err)
+		require.True(t, strings.Contains(err.Error(), dberrors.NewUnknownNamespaceError(randomNs).Error()))
```

Comment:
Is there a particular reason you can't just have an `assert.Equal()` here?

Reply:
assert v require - I use require because it fails as soon as you hit the condition (unless their are defers in the codepath, which can get complicated). Assert fails at the end of the test so its output is weirder to grok usually.

Reply:
Yeah I meant more `Equal` vs `True`, but I think the error you get back is like some long message so ignore my comment. `strings.Contains()` makes sense here.